recipe/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
 
 build:
     skip: True  # [win]
-    number: 0
+    number: 1
```

Comment:
Actually @jakirkham asked me and I recommended due to the metadata update. (You are correct that technically there is nothing new, but imagine someone downloading unpacking just to look for the maintainers :wink:)